apps/webapp/.env.example

```diff
@@ -14,4 +14,8 @@ NEXT_PUBLIC_RECAPTCHA_SITE_KEY="xxxxxxxxx"
 
 
 #  overrides env.local
-NEXT_PUBLIC_APP_ENV=prod
\ No newline at end of file
+NEXT_PUBLIC_APP_ENV=prod
+
+# Analytics and events tracking
```

Comment:
Keep your secrets secret! By adding the "NEXT_PUBLIC_" prefix, you are allowing to easy discover the env var on the client side. To protect API keys, it is better to save them on the server side, by removing the prefix.

The example key should be `g-XXXXXX` for Google ID.

The env key `NEXT_PUBLIC_APP_ENV` and `NEXT_PUBLIC_APP_GOOGLE_ID` is fine to have them like it.